tests/test_orient_sync.py

```diff
@@ -0,0 +1,72 @@
+import os
+import numpy as np
+
+from unittest import TestCase
+
+from aspire.source.simulation import Simulation
+from aspire.utils.filters import RadialCTFFilter
+from aspire.utils.preprocess import downsample
+from aspire.orientation.commonline_sync import CLSyncVoting
+
+import os.path
+DATA_DIR = os.path.join(os.path.dirname(__file__), 'saved_test_data')
+
+
+class OrientSyncTestCase(TestCase):
+    def setUp(self):
+        L = 32
+        n = 64
+        C = 1
+        pixel_size = 5
+        voltage = 200
+        defocus_min = 1.5e4
+        defocus_max = 2.5e4
+        defocus_ct = 7
+        Cs = 2.0
+        alpha = 0.1
+
+        filters = [RadialCTFFilter(pixel_size, voltage, defocus=d, Cs=Cs, alpha=alpha) for d in
+                   np.linspace(defocus_min, defocus_max, defocus_ct)]
+
+        vols = np.load(os.path.join(DATA_DIR, 'clean70SRibosome_vol.npy'))
+        vols = vols[..., np.newaxis]
+        vols = downsample(vols, (L*np.ones(3, dtype=int)))
+
+        sim = Simulation(
+            L=L,
+            n=n,
+            vols=vols,
+            C=C,
```

Comment:
Should no longer be necessary after #235 was merged.

Reply:
Remove in commit 05233c5.